index.html

```diff
@@ -5,16 +5,158 @@
+        </ul>
+    </header>
+    <main>
+        <section class="hero_image section_one">
+            <h1 class="font_anton">
+                SUCCESS
+            </h1>
+            <button class="button col-2">
+                Book Now
+            </button>
+        </section>
+        <section>
+            <section class="row">
+                <div class="col-12">
+                    <h1>Life is about creating and living experiences that are worth sharing."</h1>
+                    "We started out to get a computer in the hands of everyday people, and we succeeded beyond our
+                    wildest dreams."
+                    My job is not to be easy on people. My job is to make them better."
+
+                    "Have the courage to follow your heart and intuition. They somehow know what you truly want to
+                    become."
+
+                    "Let's go invent tomorrow rather than worrying about what happened yesterday."
+                    We don't get a chance to do that many things, and every one should be really excellent. Because
+                    this
+                    is our life. Life is brief, and then you die, you know? And we've all chosen to do this with our
+                    lives. So it better be damn good. It better be worth it."
+
+                    9. "Be a yardstick of quality. Some people aren't used to an environment where excellence is
+                    expected."
 
-    <!-- Add your markup here -->
+                    10. "You can't connect the dots looking forward; you can only connect them looking backward. So
+                    you
+                    have to trust that the dots will somehow connect in your future. You have to trust in
+                    something--your gut, destiny, life and karma, whatever. This approach has never let me down, and
+                    it
+                    has made all the difference in my life."
+                    11. "Here's to the crazy ones, the misfits, the rebels, the troublemakers, the round pegs in the
+                    square holes.... The ones who see things differently--they're not fond of rules.... You can
+                    quote
+                    them, disagree with them, glorify or vilify them, but the only thing you can't do is ignore them
+                    because they change things.... They push the human race forward, and while some may see them as
+                    the
+                    crazy ones, we see genius, because the ones who are crazy enough to think that they can change
+                    the
+                    world are the ones who do."
+                    <button class="button_about_me">
+                        Read More
+                    </button>
+                </div>
+                <div class="col-12 about_me_img">
+                    <img src="product.jpg" class="about_im_2">
+                </div>
```

Comment:
See my comment in the CSS - this image would be best as a background-image to help it scale better.